tests/db_functions/tests.py

```diff
@@ -99,6 +103,64 @@ def test_coalesce_ordering(self):
             lambda a: a.name
         )
 
+    def test_greatest(self):
+        now = timezone.now()
+        before = now - timedelta(hours=1)
+
+        Article.objects.create(
+            title="Testing with Django",
+            written=before,
+            published=now,
+        )
+
+        articles = Article.objects.annotate(
+            last_updated=Greatest('written', 'published'),
+        )
+        self.assertEqual(articles.first().last_updated, now)
+
+    @skipUnless(connection.vendor == 'postgresql', 'Postgres ignores nulls in Greatest')
```

Comment:
The default should be the most common I think. So set the default, then change it in the postgres features.